client/settings/general-settings/style.scss

```diff
@@ -0,0 +1,37 @@
+.general-settings {
+	&__bank-information {
+		display: flex;
+		flex-direction: column;
+
+		:first-child {
+			flex-grow: 1;
+		}
+
+		@include breakpoint( '>800px' ) {
+			align-items: center;
+			justify-content: space-between;
+			flex-direction: row;
+			flex-wrap: nowrap;
+		}
+	}
+
+	&__account-statement-wrapper {
+		position: relative;
+
+		.input-help-text {
+			position: absolute;
+			right: 10px;
+			top: 12px;
+			font-size: 12px;
+			color: #757575;
+
+			@include breakpoint( '>800px' ) {
```

Comment:
For widths `783-800` (incl.) this happens:

![Markup on 2021-04-30 at 22:01:02](https://user-images.githubusercontent.com/1759681/116748386-b0a09280-a9ff-11eb-99db-52e6ca660a2c.png)

Reply:
Thanks - it looks like `782` is not a breakpoint in `client/stylesheets/abstracts/_breakpoints.scss`, so it's not letting me use `@include breakpoint('>782')` - but it's definitely being used somewhere else in the `wp-admin` form styles : https://d.pr/i/5orEoY :(

I'm going to have to manually add a breakpoint here, without the `breakpoint` mixin.